### DISCUSSION TITLE
Print screen issues

### DISCUSSION THREAD

**Discussion by ahmadmuzaffarzafriy** [elishacloud/dxwrapper#166] [General]
Anyone tested dxwrapper with old games (Jane's F-15, F-18)? The screen is blank or black when print screening the game while using the wrapper...

**Comment by elishacloud** [comment]
What settings are you using?  If you are using `dd7to9` then this could be because Direct3D9 exclusive mode may not allow print screen to work.

See this full thread for more detail:  https://answers.microsoft.com/en-us/windows/forum/all/why-cant-i-shoot-or-make-printscreen-games-in/25813eb0-c170-431f-9f2f-629bb3c7515d

You can try enabling `FullscreenWindowMode` to work around this issue.
